Release.toml

```diff
@@ -59,3 +59,6 @@ version = "1.1.4"
     "migrate_v1.1.3_kubelet-cpu-manager.lz4",
 ]
 "(1.1.3, 1.1.4)" = []
+"(1.1.4, 1.1.5)" = [
```

Comment:
Shouldn't this be `1.2.0`? Since this is a new feature @jhaynes / @tjkirch?

Reply:
We should have all unreleased migrations under the same version pair so that, when we test upgrading and downgrading, all the migrations will run. In the event of a 'patch' level release, it may be necessary to edit Release.toml in the event that some migrations need to go into the patch.

As for what the unreleased version represented in Release.toml should be, I'm not sure. I don't think it matters as long as we agree (i.e. there are a few PRs open that should all agree).